### PR TITLE
Fix finalizers formatting finally

### PR DESCRIPTION
**Pull request by jrafanie**
Fix the format of the really long command output by removing some extra cruft and using a horizontal scrollbar using fancy `<code` tags.  ✨ 

Fix the publish time and date for @Fryguy without renaming the file to match.  :trollface: 

@Fryguy @imtayadeway @cybette please review

### CODE REVIEW

**Comment by cybette** [comment]
LGTM :+1:

File: site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md [hunk 1]
```diff
@@ -17,41 +17,39 @@ it's still around in the libraries we use.
...
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/nio4r-2.1.0/spec/spec_helper.rb:      sock = Timeout.timeout(1) { TCPSocket.new("localhost", $current_tcp_port) }
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/open4-1.3.4/lib/open4.rb:    Timeout.timeout(timeout){ yield }
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/rack-2.0.3/test/spec_utils.rb:    Timeout.timeout(1) do
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/ruby-dbus-0.13.0/spec/spec_helper.rb:    Timeout.timeout(10) do
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/snmp-1.2.0/lib/snmp/manager.rb:          Timeout.timeout(@timeout) do
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/vcr-3.0.3/lib/vcr/util/internet_connection.rb:          Timeout.timeout(timeout) do
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/webmock-2.3.2/spec/support/network_connection.rb:    Timeout.timeout(timeout) do
-
-05:36:21 ~/Code/manageiq (master) (2.4.2) + grep -r -E "\bTimeout::timeout\b" /Users/joerafaniello/.gem/ruby/2.4.2/gems
-/Users/joerafaniello/.gem/ruby/2.4.2/gems/fog-core-1.45.0/lib/fog/core/current_machine.rb:    #     Timeout::timeout(5) do
```
Comment (jrafanie) [inline site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: Note, I removed this commented line on purpose.  I know @Fryguy will see it.

File: site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Finalizers can be interrupted from time to time
 author: jrafanie
-date: 2017-09-21 15:36:39 UTC
+date: 2017-09-28 19:05:09 UTC
```
Comment (jrafanie) [inline site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: As requested @Fryguy

Comment (chrisarcand) [reply site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: wait whut why not rename the file...?

Comment (jrafanie) [reply site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: @cybette suggested just changing it here.  I can rename the file too. I wasn't sure how links to the blog article would be affected, such as: http://www.manageiq.org/blog/2017/09/finalizers-can-be-interrupted-from-time-to-time/

Let me know what we want to do.

Comment (jrafanie) [reply site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: Note, those links don't include the day, only the year and month so maybe we'd be fine.

Comment (chrisarcand) [reply site/_posts/2017-09-21-finalizers-can-be-interrupted-from-time-to-time.md]: yeah it doesn't really matter, whatever :)

**Comment by jrafanie** [comment]
Thanks for the feedback all!  ❤️

**Pull request merged**